src/ws-egl.cpp

```diff
@@ -180,8 +163,8 @@ bool ImplEGL::initialize(EGLDisplay eglDisplay)
     m_eglDisplay = eglDisplay;
 
     /* Initialize Linux dmabuf subsystem. */
-    if (isEGLExtensionSupported(extensions, "EGL_EXT_image_dma_buf_import")
-        && isEGLExtensionSupported(extensions, "EGL_EXT_image_dma_buf_import_modifiers")) {
+    if (epoxy_has_egl_extension(eglDisplay, "EGL_EXT_image_dma_buf_import")
+        && epoxy_has_egl_extension(eglDisplay, "EGL_EXT_image_dma_buf_import_modifiers")) {
         s_eglQueryDmaBufFormatsEXT = reinterpret_cast<PFNEGLQUERYDMABUFFORMATSEXTPROC>(eglGetProcAddress("eglQueryDmaBufFormatsEXT"));
```

Comment:
When using libepoxy we do not need to use `eglGetProcAddress()` by hand for these functions either:

```
% grep 'define eglQueryDmaBuf' /usr/include/epoxy/* 
/usr/include/epoxy/egl_generated.h:#define eglQueryDmaBufFormatsEXT epoxy_eglQueryDmaBufFormatsEXT
/usr/include/epoxy/egl_generated.h:#define eglQueryDmaBufModifiersEXT epoxy_eglQueryDmaBufModifiersEXT
```

Reply:
:+1: